pkgs/development/python-modules/openbabel-bindings/default.nix

```diff
@@ -13,7 +13,7 @@ buildPythonPackage rec {
   # some functionality to work (inparticular, pybel).
   # see https://openbabel.org/docs/dev/Installation/install.html
   BABEL_LIBDIR = "${openbabel}/lib/openbabel/3.1.0";
-  LD_LIBRARY_PATH = "$LD_LIBRARY_PATH:${openbabel}/lib";
+  LD_LIBRARY_PATH = "${openbabel}/lib";
```

Comment:
The previous version just put the literal unexpanded string `$LD_LIBRARY_PATH` into the `LD_LIBRARY_PATH` variable, which wasn’t useful.

```console
$ nix-shell -A python3Packages.openbabel-bindings --run 'echo "$LD_LIBRARY_PATH"'
$LD_LIBRARY_PATH:/nix/store/hc33pncm3ggml00kay5h64fkivxnsdni-openbabel-3.1.1/lib
```